config/label_templates.yml

```diff
@@ -105,3 +105,9 @@ templates:
     :printer_type: 96 Well Plate
     :pmb_template: sqsc_96plate_label_template_code39
     :sprint_template: plate_96_2d.yml.erb
+
+  kinnex_tube:
+    :label_class: Labels::TubeLabelKinnex
+    :printer_type: 96 Well Plate
+    :pmb_template: sqsc_96plate_label_template_code39
+    :sprint_template: plate_96_2d.yml.erb
```

Comment:
I think you should send the photo of the label to the lab stakeholders to ask whether this is the correct type of label. I think we're transferring into 'normal' tubes so imagine we should be using a 'tube' label rather than a 'plate' label. If that means we can't have a 2D barcode then that's probably OK, I guess they can reprint from Traction.